ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverter.java

```diff
@@ -19,15 +20,16 @@ public class EventStreamResponseConverter {
 
 	public static final String CUSTOM = "http://example.org/";
 	public static final Property MEMBER_TYPE = createProperty(CUSTOM, "memberType");
+	private final String hostname;
 	private final ViewSpecificationConverter viewSpecificationConverter;
 
-	public EventStreamResponseConverter(ViewSpecificationConverter viewSpecificationConverter) {
+	public EventStreamResponseConverter(AppConfig appConfig, ViewSpecificationConverter viewSpecificationConverter) {
 		this.viewSpecificationConverter = viewSpecificationConverter;
+		hostname = appConfig.getHostName();
 	}
 
 	public EventStreamResponse fromModel(Model model) {
-		final String collection = getIdentifier(model, createResource(EVENT_STREAM_TYPE)).orElseThrow()
-				.replace(LDES, "");
+		final String collection = getIdentifier(model, createResource(EVENT_STREAM_TYPE)).orElseThrow().getLocalName();
```

Comment:
This can result in an NPE. `getLocalName` returns null when the resource is not a URI. I think it would be better to insert `.map(Resource::getLocalName)` before the `orElseThrow`. Then maybe throw a clear exception instead of generic one?